examples/upload-vmck-json.sh

```diff
@@ -0,0 +1,3 @@
+nomad job run -output vmck.nomad > vmck.json # convert hcl to json file for upload on server TODO: custom nomand and json filename
```

Comment:
Please add `#!/bin/bash -ex` as the first line, otherwise you can't run the script as `./examples/upload-vmck-json.sh`